ee/modules/021-cni-cilium/templates/egress-gateway-agent/daemonset.yaml

```diff
@@ -42,6 +42,8 @@ spec:
       maxUnavailable: 1
   template:
     metadata:
+      annotations:
+        {{ include "helm_lib_ds_eviction_annotation" . | nindent 8 }}
       labels:
```

Comment:
Don't we need it here?
https://github.com/deckhouse/deckhouse/blob/main/modules/021-cni-cilium/templates/_agent-daemonset.tpl

Reply:
https://github.com/deckhouse/deckhouse/pull/8609/files#diff-2972bb64653a0104aeaa2ce62adfa53bc748389967677ca2cb23407027388516